include/mitsuba/render/imageblock.h

```diff
@@ -140,6 +140,9 @@ class MTS_EXPORT_RENDER ImageBlock : public Object {
      */
     Mask put(const Point2f &pos, const Float *value, Mask active = true);
 
+    /// Overwrite entire channel with the given value.
+    void overwrite_channel(size_t channel, ScalarFloat value);
```

Comment:
Can we remove this interface? Mitsuba 1 did not need this to implement the particle tracer.